codestodians/food-basket-front-end
# [Feature: #155333907] Transfer Logic for the Header into its container component.

#### What does this PR do?
- Transfers all logic (handlers majorly) into the `Header` container component

#### Description of Task to be completed?
- Abstract portion of the Header component.
- Refactor Header connect component
- Update test specs

#### How should this be manually tested?
- Run the app with `yarn start`.
- Run tests with `yarn test`
- Everything should work as before.

#### Any background context you want to provide?
- As long as a component is separated into presentational and container components, it only makes sense for each to do only what it is supposed to do. Presentational: render JSX; Container: Handle component logic. This PR takes care of that for the Header component.

#### What are the relevant pivotal tracker stories?
[#155333907](https://www.pivotaltracker.com/story/show/155333907)
